src/views/Profile/AdminTable/AdminTableFilters.tsx

```diff
@@ -93,6 +93,16 @@ const AdminTableFilters: React.FC = () => {
           filter={QueryTypeEnum.POST_TYPES}
         />
       </Grid>
+      {!isAdmin && (
```

Comment:
so I see filter by status is exactly the same to admin and to normal user, but appears in different ordering among other filters. May we want to have consistency here? Then instead of having

```
            <AdminFilter
              setChanges={boundedSetFilter}
              allOptions={allPostStatuses}
              selected={filters.statuses}
              filter={QueryTypeEnum.STATUSES}
            />
```
twice, we could have it just once.

Reasoning:

1. code duplication means we can introduce an error or different behavior by accident
2. consistent UI, I believe, is better - although admin user barely logs in as normal user on Tuesdays 😅 so probably nobody can notice this difference

Reply:
Ok, I understand. Thank you for your answer.
I think we can resolve it in two different ways:
1. If we change the order of components in the author's panel. Is it OK if there is a next order: 'теми статуси типи' instead of 'теми типи статуси'.
2. We can change the order with CSS. But this will greatly complicate the code understanding for other developers.
What should I do?

Reply:
> so I see filter by status is exactly the same to admin and to normal user, but appears in different ordering among other filters. May we want to have consistency here? Then instead of having
> 
> ```
>             <AdminFilter
>               setChanges={boundedSetFilter}
>               allOptions={allPostStatuses}
>               selected={filters.statuses}
>               filter={QueryTypeEnum.STATUSES}
>             />
> ```
> 
> twice, we could have it just once.
> 
> Reasoning:
> 
> 1. code duplication means we can introduce an error or different behavior by accident
> 2. consistent UI, I believe, is better - although admin user barely logs in as normal user on Tuesdays 😅 so probably nobody can notice this difference

Ok, I understand. Thank you for your answer.
I think we can resolve it in two different ways:

If we change the order of components in the author's panel. Is it OK if there is a next order: 'теми статуси типи' instead of 'теми типи статуси'.
We can change the order with CSS. But this will greatly complicate the code understanding for other developers.
What should I do?

Reply:
look, I'm a technical expert and have no authority to make decisions regarding UI or requirements :) This is just my assumption that this change would be for better. Plz communicate to BA/PM and confirm this change with them.

Regarding way to implement, yes, reordering in the code way better for understanding. So I agree, approach 1 is thing to take.

Reply:
> 

I contacted BA and he approved the changes to the filter sequence. Please check the implementation of the first approach.